can-value.js

```diff
@@ -1 +1,24 @@
-module.exports = {};
+var CanKey = require("can-key");
+var KeyObservable = require("can-simple-observable/key/key");
+var Observation = require("can-observation");
+var SetterObservable = require("can-simple-observable/setter/setter");
+
+module.exports = {
+	bind: function(object, keyPath) {
+		return new KeyObservable(object, keyPath);
+	},
+
+	from: function(object, keyPath) {
+		return new Observation(function() {
+			return CanKey.get(object, keyPath);
+		});
+	},
+
+	to: function(object, keyPath) {
+		return new SetterObservable(function() {
+			// This line intentionally left blank :)
```

Comment:
Is this the right way to set up this observer?

Reply:
I would just create what you need:

```
return assignSymbols({},{
  setValue: function(newValue){
    key.set(object, keyPath, newValue)
  }
})
```

You'll need to make sure the mutated by stuff is setup too.

Reply:
creating a `setterObservable` is overkill.  We don't need the underlying observable at all.